modules/services/window-managers/bspwm/default.nix

```diff
@@ -58,13 +58,17 @@ in {
       ${concatStringsSep "\n" (mapAttrsToList formatRule cfg.rules)}
 
       # java gui fixes
-      export _JAVA_AWT_WM_NONREPARENTING=1
```

Comment:
Why not keep it in both places (for those using the resulting configuration without `xsession.enable = true`)?

Reply:
Good point